bitnami/etcd/templates/statefulset.yaml

```diff
@@ -164,11 +166,11 @@ spec:
         - name: ETCD_DATA_DIR
           value: {{ template "etcd.dataDir" . }}
         - name: ETCD_ADVERTISE_CLIENT_URLS
-          value: "{{ $etcdClientProtocol }}://$(MY_POD_NAME):{{ .Values.service.port }}"
+          value: "{{ $etcdClientProtocol }}://$(MY_POD_NAME).{{ $etcdHeadlessServiceName }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.service.port }}"
```

Comment:
Why do you still need `$(MY_POD_NAME)`?

Reply:
I suppose because it is using headless services

Reply:
because it is different for each replica

etcd-0, etcd-1, etcd-2,...

Reply:
👍